### PR TITLE
 <Table/> - add UniDriver

### PR DESCRIPTION
**Pull request by Lukermelesh**

### CODE REVIEW

File: src/DataTable/DataTable.uni.driver.js [hunk 1]
```diff
@@ -77,15 +77,15 @@ export const dataTableUniDriverFactory = base => {
      * Get header cell element: (columnIndex) => Element
      * @deprecated Should be private
      */
-    getHeaderCell: index => getHeaderCell(index).getNative(), // eslint-disable-line no-restricted-properties
```
Comment (Lukermelesh) [inline src/DataTable/DataTable.uni.driver.js]: @mosheKerbel I don't see any reason to use `getNative()` here and below.

Comment (mosheKerbel) [reply src/DataTable/DataTable.uni.driver.js]: problem is that consumers can still use this functions, and if they do then it's a breaking change.
should revert.

File: src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.uni.driver.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
 import { baseUniDriverFactory } from 'wix-ui-test-utils/base-driver';
 import styles from './MessageBoxFunctionalLayout.scss';
-import { ReactBase } from '../../../test/utils/unidriver';
-
-const getElement = async elem => ((await elem.exists()) ? elem : null);
```
Comment (mosheKerbel) [inline src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.uni.driver.js]: 👍

File: src/MultiSelectCheckbox/MultiSelectCheckbox.spec.js [hunk 3]
```diff
@@ -71,9 +71,7 @@ describe('multiSelectCheckbox', () => {
     { key: 'Space', keyCode: 32, which: 32 },
   ];
   OPEN_DROPDOWN_CHARS.forEach(charData => {
-    it(`should show dropdown on input focus and press on ${
-      charData.key
-    }`, () => {
+    it(`should show dropdown on input focus and press on ${charData.key}`, () => {
```
Comment (mosheKerbel) [inline src/MultiSelectCheckbox/MultiSelectCheckbox.spec.js]: looks like an eslint change and it's causing you conflicts.

File: testkit/index.js [hunk 4]
```diff
@@ -139,6 +139,7 @@ export const sliderTestkitFactory = testkitFactoryCreator(load(require('../src/S
 export const sortableListTestkitFactory = testkitFactoryCreator(load(require('../src/SortableList/SortableList.driver')));
 export const statsWidgetTestkitFactory = testkitFactoryCreator(load(require('../src/StatsWidget/StatsWidget.driver')));
 export const tableTestkitFactory = testkitFactoryCreator(load(require('../src/Table/Table.driver')));
+export const tableTestkitUniFactory = uniTestkitFactoryCreator(load(require('../src/Table/Table.uni.driver')));
```
Comment (mosheKerbel) [inline testkit/index.js]: this file is auto generated so this change is irrelevant, should b rereverted.

File: src/Table/Table.private.driver.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import tableDriverFactory from './Table.driver';
+
+export const tablePrivateDriverFactory = args => {
+  const publicDriver = tableDriverFactory(args);
+  return {
+    ...publicDriver,
+    exists: () => !!publicDriver.element,
+    getInnerHtml: () => publicDriver.element.innerHTML,
```
Comment (mosheKerbel) [inline src/Table/Table.private.driver.js]: i would expect to this function also in a private uni.driver and not in the public one.

File: src/Table/Table.private.driver.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    getCellTextAt: (rowIndex, cellIndex) =>
+      publicDriver.getCell(rowIndex, cellIndex).textContent,
```
Comment (mosheKerbel) [inline src/Table/Table.private.driver.js]: i would expect to this function also in a private uni.driver and not in the public one.

File: src/Table/Table.private.driver.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import tableDriverFactory from './Table.driver';
+
+export const tablePrivateDriverFactory = args => {
+  const publicDriver = tableDriverFactory(args);
+  return {
+    ...publicDriver,
+    exists: () => !!publicDriver.element,
+    getInnerHtml: () => publicDriver.element.innerHTML,
+    getCellTextAt: (rowIndex, cellIndex) =>
```
Comment (mosheKerbel) [inline src/Table/Table.private.driver.js]: i would expect to this function also in a private uni.driver and not in the public one.

**Comment by mosheKerbel** [comment]
@Lukermelesh 
http://pullrequest-tc.dev.wixpress.com/viewLog.html?buildId=3220046&buildTypeId=FedInfra_WixStyleReact
see that table.spec.js is failing

**Comment by mosheKerbel** [comment]
bulid is failng due to 1 flaky unrelated test, merging.

**Pull request merged**